### DISCUSSION TITLE
Why would I use clerk over emacs org mode?

### DISCUSSION THREAD

**Discussion by port19x** [nextjournal/clerk#308] [Q&A] [answered]
I consider myself an intermediate clojure dev, but have used emacs long before clojure so I have a bias in favor of [org mode](https://orgmode.org/).

If I understand correctly, clerk is just another way to accomplish literate programming with it's usual perks:
- version controllable
- markdown support
- executable code blocks
- incremental execution
- html export

Is there some deeper integration, like perhaps some profiling or debugging capabilities, that clerk comes with that would warrant a user like me taking it for a spin?
What are the rich builtin-viewers reffering too? Is this just a weird way to say I can add images?
Because if it's just another code block container I find it hard to see a point.

**Comment by jackrusher** [comment, marked as answer]
You might have a look at the [demos](https://github.clerk.garden/nextjournal/clerk-demo/commit/3ccf0697a44908458d13485fbaf2ff680f7f25de/), which contains many examples of viewers in action.
